<launch>

	<node pkg="fast_livo" type="stamp_calib" name="stamp_calib_node" output="screen" >
	</node>


	<node pkg="image_transport" type="republish" name="republish" args="compressed in:=/usb_cam/image_raw raw out:=/usb_cam/image_raw" output="screen" respawn="true"/>

launch-prefix="gdb -ex run --args" launch-prefix="valgrind --leak-check=full --show-leak-kinds=all"
</launch>